### PR TITLE
[W09H01] fix missing import & better cross platform compatibility

### PR DESCRIPTION
**Pull request by felixhauptmann**
* add missing import
* better cross platform support (replace `'\n'` with `System.lineSeparator()`)

### CODE REVIEW

File: w09h01/test/pgdp/pingutrip/PrettyDirectionsTest.java [hunk 1]
```diff
@@ -18,6 +18,6 @@ void testIsOnWayTest() {
         assertEquals("""
                 1 Schritte Richtung 0 Grad.
                 3 Schritte Richtung 0 Grad.
-                2 Schritte Richtung 45 Grad.""", output);
+                2 Schritte Richtung 45 Grad.""".replace("\n", System.lineSeparator()), output);
```
Comment (Anatoly03) [inline w09h01/test/pgdp/pingutrip/PrettyDirectionsTest.java:21]: That’s very caring of you!

File: w09h01/test/pgdp/pingutrip/ReadWayPointsTest.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package pgdp.pingutrip;
 
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```
Comment (Anatoly03) [inline w09h01/test/pgdp/pingutrip/ReadWayPointsTest.java:4]: How could _that_ happen? 🫣

Comment (felixhauptmann) [reply w09h01/test/pgdp/pingutrip/ReadWayPointsTest.java:4]: ask @nilsreichardt :)

(no, ping wasn't unnecessary - you can merge)

**Comment by felixhauptmann** [comment]
@nilsreichardt can get merged as well

**Review by nilsreichardt — APPROVED** [review]
LGTM

**Pull request merged**